docs/_writing-tests/h2responsewriter.md

```diff
@@ -0,0 +1,113 @@
+## Response Writer API ##
+
+##### `write_headers(self, headers, status_code, status_message=None, stream_id=None, last=False):`
```

Comment:
This header level seems very low; is there prededent somewhere else in the docs?

Reply:
As far as I can see, there is no precedent for this kind of API documentation. There are some js API docs but those are mostly showing a block of code and explaining what they do by the looks of it. I can bump it up a level or two?

Reply:
Yeah, see if that looks better.